projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1020 @@
+```
+
+### created
+
+UiTdatabank keeps track of the creation dates of each document. This metadata for the creation date can be queried with the `created` parameter.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+> Note that the `+` sign should be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Get all events and places that are created since the start of 2022:
+```
+GET /offers/?q=created:[2022-01-01T00\:00\:00%2B01\:00 TO *]
+```
+
+### creator
+
+Use the `creator` parameter to search for documents created by a specific creator.
+
+<!-- theme: warning --> 
+> At the time of writing, creator can contain a nickname or emailaddress. In the near future creator will only contain the uuid.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any uuid of a creator.
+
+**Example**
+
+Retrieve all events and places craeted by user with uuid `86a02c65-696d-5b12-a9b1-9e3bc8e6303c`:
+
+```
+GET /offers/?q=creator:86a02c65-696d-5b12-a9b1-9e3bc8e6303c
+```
+
+### dateRange
+
+Using the `dateRange` parameter, you can get all events and places that took place in a given range in the past, or will take place in a given range in the future.
+
+> By default, the search API will only return results that are currently available. In order to also retrieve results that are not available (yet), you'll need to disable the default filters for `availability`. You can reset this default as described in the Default Filters (TO DO: link to guide).
+
+**Applicable on endpoints** 
+
+`/events` `/places` `/offers` 
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+> Note that the `+` sign should be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Retrieve all events that took place in the year 2020:
+```
+GET /events/?q=dateRange:[2020-01-01T00\:00\:00%2B01\:00 TO 2021-12-31T23\:59\:59%2B01\:00]&availableTo=*&availableFrom=*
+```
+
+### \_exists_
+
+With the `_exists_` parameter you can search for the documents that have a specific property in their JSON body.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any property name from the UiTdatabank JSON scheme.
+
+**Example**
+
+Retrieve all events with a description in dutch:
+
+```
+GET /events/?q=_exists_:description.nl
+```
+
+### id
+
+Retrieve events that match a specific uuid with the `id` parameter. It is possible to search by an event id, place id or organizer id.
+
+A specific id can be found by
+-   looking for the value for the `id` property
+```js
+{
+   "id": "75573a64-ddc8-4fd0-8b07-d258939dd74f"
+}
+```
```

Comment:
In what scenario is this? Our events, places and organizers currently do not have an `id` property.

Reply:
Not in the JSON of the event or in Search API. However, if you create an event / place / organizer / image or mediaObject / video through the API the response will look like this: 

```
{
    "id": "affcc3d4-95bb-46cf-8095-cde4d755f351",
    "placeId": "affcc3d4-95bb-46cf-8095-cde4d755f351",
    "url": "https://io-acc.uitdatabank.be/place/affcc3d4-95bb-46cf-8095-cde4d755f351",
    "commandId": "00000000-0000-0000-0000-000000000000"
}
```

But I see this is not clear in the documentation, so it needs some rework

Reply:
I understand it now based on your comment but I think it's still unclear in the documentation. I think it would be clear if we reword:

> looking for the value for the `id` property in the JSON response of the UiTdatabank entry API

to:

> looking for the value for the `id` property in the JSON response of the UiTdatabank entry API to a `POST /events` request (to create a new event)